### PR TITLE
changing supported node version verbiage

### PR DESCRIPTION
**Pull request by tylercherpak**
### Description of the Change

Changing supported node version verbiage.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: _includes/markdown/Installation.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Prerequisites
 
 WP Local Docker requires docker, docker-compose, Node, and npm. It is recommended that you use the latest versions of
-docker and docker-compose. Node 10 (current LTS version) is the only version of node that is supported. While WP Local
+docker and docker-compose. Node 10 is the only version of node that is currently supported. While WP Local
```
Comment (eugene-manuilov) [inline _includes/markdown/Installation.md:4]: I think we can make it better:

```suggestion
docker and docker-compose. Node 10 is the minimum version of node that is currently supported. While WP Local
```

Comment (dustinrue) [reply _includes/markdown/Installation.md:4]: I like this change

Comment (TheLastCicada) [reply _includes/markdown/Installation.md:4]: +1

**Comment by tylercherpak** [comment]
Thanks @eugene-manuilov !

**Comment by jeffpaul** [comment]
@eugene-manuilov you good for this to get merged?

**Comment by eugene-manuilov** [comment]
@jeffpaul  not yet, we need to update the version. @tylercherpak can you update it to be `12`? `10` isn't enough for `2.8.0`.

**Comment by jeffpaul** [comment]
@eugene-manuilov just made that Node version update, we good to merge this in now?

**Comment by eugene-manuilov** [comment]
@jeffpaul yep, looks good to me

**Pull request merged**